cleverhans_tutorials/mnist_blackbox.py

```diff
@@ -285,6 +286,13 @@ def mnist_blackbox(train_start=0, train_end=60000, test_start=0,
 
 
 def main(argv=None):
+  # Warn user if running cleverhans from a different directory than tutorial.
+  cur_dir = os.path.split(os.path.dirname(os.path.abspath(__file__)))[0]
+  ch_dir = os.path.split(cleverhans.__path__[0])[0]
+  if cur_dir != ch_dir:
+    warnings.warn("Rnning cleverhans from a different directory than "
```

Comment:
I'd suggest a longer message with more explanation.

`It appears that you have at least two versions of cleverhans installed, one at %(cur_dir)s and one at %(ch_dir)s. You are running the tutorial script from the former but python imported the library module from the latter. This may cause errors, for example if the tutorial version is newer than the library version and attempts to call new features.`